story/.storybook/preview.js

```diff
@@ -0,0 +1,15 @@
+export const parameters = {
+  backgrounds: {
+    default: 'short',
+    values: [
+      {
+        name: 'white',
+        value: '#fff'
+      },
+      {
+        name: 'short',
+        value: '#fafafa'
+      }
+    ]
+  }
+};
```

Comment:
I wonder what are those?

Reply:
These are the parameters for storybook backgrounds https://storybook.js.org/docs/react/essentials/backgrounds